CHANGES.rst

```diff
@@ -6,7 +6,7 @@
 ==================
 
 - Add zope.lifecycleevent subscribers. You must include ``subscribers.zcml``
-  to use these. See https://github.com/zopefoundation/zc.intid/issues/5
+  to use these and have :mod:`zope.intid` installed. See :issue:`5`.
```

Comment:
It's a [configured](https://github.com/zopefoundation/zc.intid/blob/master/doc/conf.py#L355) usage of the extlink extension. I've been using it in gevent to great effect (but then the gevent changelog contains *a lot* of links to issue/prs).